repo/packages/D/dynatrace/1/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "packagingVersion": "3.0",
+    "name": "dynatrace",
+    "version": "latest",
```

Comment:
`version` should be `baseTechVersion-integrationVersion`

Reply:
sounds good

Reply:
done